libs/payments/ui/src/lib/client/components/PaymentSection/index.tsx

```diff
@@ -30,6 +37,24 @@ export function PaymentSection({
   cart,
   locale,
 }: PaymentFormProps) {
+  const params = useParams();
```

Comment:
Thinking of moving this change to `<MetricsWrapper>` in `layout.tsx` once https://github.com/mozilla/fxa/pull/17725 is merged

Reply:
PR is merged, this can be combined if desired! I agree that moving some control of the glean into `MetricsWrapper` is a good idea